app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.js

```diff
@@ -106,6 +106,20 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
       return (withdrawn === 'withdrawn') ? 'true' : 'false'
     },
 
+    getTaxonomyAll: function () {
```

Comment:
Optimisation suggestion: could these two new functions be one function?

```JavaScript
taxonomy_all: this.splitLongThing('taxon-slugs'),
taxonomy_all_ids: this.splitLongThing('taxon-ids'),

...

splitLongThing: function (metatag) {
  var tag = this.getMetaContent(metatag)
  if (tag) {
    return window.GOVUK.analyticsGa4.core.trackFunctions.splitStringIntoParts(tag)
  }
}
```

Reply:
@andysellick Good idea - thanks :+1: